documentacion/posintegrado/README.md

```diff
@@ -0,0 +1,275 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado, cuenta de 2 partes, la primera pieza es una librería/SDK escrito en C,
+el cual es multi plataforma, y debe ser instalado en tu maquina para poder ser utilizado por la
+segunda pieza, la cual mediante un Wrapper, accede a las funciones disponibles en la librería C.
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
+puerto serial o adaptador USB Serial.
+
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tu eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```

Comment:
```suggestion
Tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```

Reply:
Maybe we should add some examples or information on what driver is known to work with what adapters?